assets/src/scss/new-identity/style.scss

```diff
@@ -74,6 +74,9 @@
   --nav-link--visited--color: var(--grey-900);
   --nav-search-clear--background-color: var(--grey-900);
   --top-navigation--separation: 1px solid var(--grey-300);
+
+  // TODO: Review these fallback variables after we'll merge the new identity colors
+  --comments-block--comment-respond--background: var(--color-background-comments_block);
```

Comment:
This variable no longer exists since you added the `.form-section` back into the selector, it's now `--comments-block--form-section--comment-respond--background` which is very long 😬 We could either use this or define another name